etc/docker/test/matrix_policy_package_tests.yml

```diff
@@ -1,46 +1,28 @@
-atlas:
+belleii:
```

Comment:
@bari12 do we want to replace atlas vo-tests with belleii?

Reply:
Hmm not really, @cserf what's the motivation here?